cheroot/connections.py

```diff
@@ -0,0 +1,254 @@
+"""Utilities to manage open connections."""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import io
+import select
+import six
+import socket
+import time
+
+from . import errors
+from .makefile import MakeFile
+
+from collections import OrderedDict
```

Comment:
this is an import from stdlib, it should go into the top group of imports

Reply:
Done.